spec/request/swagger_spec.rb

```diff
@@ -1319,6 +1319,19 @@
           )
         end
       end
+
+      it 'supports posting to initialize a vet360_id' do
+        expect(subject).to validate(:post, '/v0/profile/initialize_vet360_id', 401)
+
+        VCR.use_cassette('vet360/person/init_vet360_id_success') do
```

Comment:
`VCR:MATCH_EVERTHING` ?

Reply:
Good call

Reply:
I like it in our request specs, but not in the overarching swagger spec.  There is no precedence for it, and IMO the important matching checks happen at the controller request spec level (i.e. `spec/request/vet360/person_request_spec.rb`).